demos/main.js

```diff
@@ -4,74 +4,65 @@
 (function () {
     'use strict';
 
-    function appendProtocol(url) {
-        return window.location.protocol + url;
-    }
-
-    // default source, can be overriden by URL
-    var default_tile_source = 'mapzen',
-        rS;
-
-    var tile_sources = {
-        'mapzen': {
-            source: {
+    var
+        tile_sources = {
+            mapzen: {
                 type: 'GeoJSONTileSource',
-                url:  appendProtocol('//vector.mapzen.com/osm/all/{z}/{x}/{y}.json')
```

Comment:
I think we were doing this to properly support https iframing? It's annoying and messy, but will need to make sure we didn't break anything.